defender/wallet-migrator/approveTokens.js

```diff
@@ -0,0 +1,143 @@
+require('dotenv').config();
+const yaml = require('js-yaml');
+const fs = require('fs');
+const axios = require('axios');
+const { ethers } = require('ethers');
+const { RelayClient } = require('defender-relay-client');
+// grab secrets from .env file
+const GOERLI_RPC_URL = process.env.GOERLI_RPC_URL;
+const MAINNET_RPC_URL = process.env.MAINNET_RPC_URL;
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+// grab Defender API Key, Secret Key and Covalent API Key from .secrets/dev.yml file
+// we need to replace the "-" from yaml files, or it throws an error when importing
+let secretsFile = yaml.load(fs.readFileSync('defender/.secrets/dev.yml', 'utf8', { schema: 'JSON_SCHEMA' }));
+secretsFile = JSON.parse(JSON.stringify(secretsFile).replace(/-/g, ''));
+const DEFENDER_API_KEY = secretsFile.keys.defenderapikey;
+const DEFENDER_API_SECRET = secretsFile.keys.defenderapisecret;
+const COVALENT_API_KEY = secretsFile.keys.covalentapikey;
```

Comment:
```suggestion
const DEFENDER_API_KEY = secretsFile.keys['defender-api-key'];
const DEFENDER_API_SECRET = secretsFile.keys['defender-api-secret'];
const COVALENT_API_KEY = secretsFile.keys['covalent-api-key'];
```
You grab the values directly like this. Parsing is not needed.